website/docs/docs/cloud/connect-data-platform/about-connections.md

```diff
@@ -20,9 +20,13 @@ dbt Cloud can connect with a variety of data platform providers including:
 - [Starburst or Trino](/docs/cloud/connect-data-platform/connect-starburst-trino)
 - [Teradata](/docs/cloud/connect-data-platform/connect-teradata) <Lifecycle status="Preview" />
 
-You can connect to your database in dbt Cloud by clicking the gear in the top right and selecting **Account Settings**. From the Account Settings page, click **+ New Project**.
+You can connect to your database in dbt Cloud by:
```

Comment:
You can leave this as is, but the following instructions would be "clicking" if you read right after. It might flow a little better like this
```suggestion
To connect to your database in dbt Cloud:
```